hash/c64_cass.xml

```diff
@@ -4983,6 +5140,36 @@ Some are based on the C64 Tape Archive at https://archive.org/download/c64tapes/
 		</part>
 	</software>
 
+	<software name="flightpa">
+		<description>Flight Path 737</description>
+		<year>1984</year>
+		<publisher>Anirog</publisher>
+
+		<part name="cass1" interface="cbm_cass">
+			<dataarea name="cass" size="610450">
+				<rom name="Flight_Path_737-_Advanced_Pilot_Trainer.tap" size="610450" crc="97f34bcf" sha1="b8888c5b9855e8492d39f3ecc23eabf357a83a98"/>
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="flintstn" supported="no"> <!-- game loads but doesn't run (just displays a light blue screen) -->
+		<description>The Flintstones</description>
+		<year>1989</year>
+		<publisher>Grandslam</publisher>
+
+		<part name="cass1" interface="cbm_cass">
+			<dataarea name="cass" size="614404">
+				<rom name="Flintstones,_The.tap" size="614404" crc="e4a2b051" sha1="b6490b868ce4c76addec347cdc0b8ecf2a325ac3"/>
+			</dataarea>
+		</part>
+
+		<part name="cass2" interface="cbm_cass">
+			<dataarea name="cass" size="614404">
+				<rom name="Flintstones,_The_a1.tap" size="614404" crc="ea60184b" sha1="c82606fd67a85b210faef3ac05fe4837002e5662"/>
```

Comment:
You’re still using edited labels here.  Please just make the labels match the labels on the tapes as closely as possible.  Don’t do things like moving articles to the end  like you’ve done here.

Reply:
These labels are as per the files found within Ultimate Tape Archive V2 set.  I haven't changed a thing.  The "_a1" at the end is the alternative tape image which is typically Side 2/Side B of the same cassette but containing an identical copy of the program that is found on Side 1/Side A

Reply:
I haven't got to the games starting with H yet.  It didn't take me long to find a similar example.  The existing entry for "hoppinmad" has been done in the same manner with the alternative file ending with "_a1".

Reply:
can confirm that this is how the sources are named, so scoomby is doing the right thing here.

unless you're seriously saying we should have 2 files with identical names in the root folder of a zip file, which it only likely to cause human error etc. when dealing with such things.